versions.lock

```diff
@@ -29,7 +29,7 @@ com.google.cloud:google-cloud-core:1.94.3 (3 constraints: da2307c0)
 com.google.cloud:google-cloud-core-http:1.94.3 (2 constraints: 651543d8)
 com.google.cloud:google-cloud-storage:1.113.14 (2 constraints: 10143aa3)
 com.google.code.findbugs:annotations:3.0.1 (1 constraints: 0605fb35)
-com.google.code.findbugs:jsr305:3.0.2 (9 constraints: 057f5545)
+com.google.code.findbugs:jsr305:3.0.2 (2 constraints: cd195721)
```

Comment:
What are the remaining 2 references?

Reply:
I excluded it from ref-guide and s3-repository as well, and now we're down to only

```
./gradlew why --hash=b70f3485
> Task :why
com.google.code.findbugs:jsr305:3.0.1
        com.google.code.findbugs:annotations -> 3.0.1
```

I think this one is the testOnly from https://github.com/apache/solr/blob/main/gradle/hacks/solr.findbugs.gradle#L40